src/types/rest-types.ts

```diff
@@ -122,6 +122,12 @@ export type RestAPIEndpoints = FhirAPIEndpoints & {
 
   'GET /v1/fhir/files/projects/:projectId/files/:fileId': {
     Request: {
+      /**
+       * If accessing a file owned by a subject, it is recommended to
+       * specify the subject id here.
+       */
```

Comment:
Why is it "recommended" and not enforced?

Reply:
Because someone with a very permissive policy won't necessarily need it.